build.gradle

```diff
@@ -20,10 +23,35 @@ gradle.projectsEvaluated {
   }
 }
 
+task checkstyle(type: Checkstyle) {
+  description 'Check code standard'
+  group 'verification'
+  configFile file('config/checkstyle.xml')
+  source 'src'
+  classpath = files()
+  ignoreFailures = false
+}
+
+task pmd(type: Pmd) {
+  ruleSetFiles = files('config/pmd.xml')
+  ruleSets = []
```

Comment:
It's an empty array because otherwise it'll ignore the `config/pmd.xml` that has our config.